### PR TITLE
chore: enhance reliability on visual regression tests

### PR DESCRIPTION
**Pull request by tujoworker**
In addition to the PR, I have experimented a little with using webkit and chromium in order to gain the snapshot picture quality. 
But found, when ruining the tests on CI, that they need a higher failure threshold than firefox.

### CODE REVIEW

**Comment by langz** [comment]
Great work, the code looks good, but when I ran `yarn test:screenshots:update` locally now, I got quite a few changes:
```
Test Suites: 4 failed, 54 passed, 58 total
Tests:       14 failed, 449 passed, 463 total
Snapshots:   32 updated, 419 passed, 451 total
```

image


I'll give it one more try, and see if I get the same result.

**Comment by langz** [comment]
> @langz Could you retry it on your machine? Remember to run `yarn` (install) first.

I will give it a spin🕸️

**Comment by langz** [comment]
> > @langz Could you retry it on your machine? Remember to run `yarn` (install) first.
> 
> I will give it a spin🕸️

After testing now, it seems much better here.

Here's the summary:
```
Snapshot Summary
 › 1 snapshot updated from 1 test suite.

Test Suites: 2 failed, 56 passed, 58 total
Tests:       2 failed, 461 passed, 463 total
Snapshots:   1 updated, 462 passed, 463 total
Time:        403.268 s, estimated 405 s
```


I do however seem to get two tests that fail with "Transport not open", as reported earlier:

```
 FAIL   browser: firefox  src/components/drawer/__tests__/Drawer.screenshot.test.ts (14.412 s)
  ● Drawer › open › have to match correct scroll view setup

    Transport not open

      at send (http:/localhost:8000/socket.io/socket.io.js:3135:15)
      at flush (http:/localhost:8000/socket.io/socket.io.js:2801:24)
      at sendPacket (http:/localhost:8000/socket.io/socket.io.js:2867:12)
      at write (http:/localhost:8000/socket.io/socket.io.js:2821:12)
      at _packet (http:/localhost:8000/socket.io/socket.io.js:598:21)
      at packet (http:/localhost:8000/socket.io/socket.io.js:1001:15)
      at emit (http:/localhost:8000/socket.io/socket.io.js:981:14)
      at unregisterPath (webpack-internal:/.cache/socketIo.js:63:10)
      at getDerivedStateFromProps (webpack-internal:/.cache/query-result-store.js:97:64)
      at applyDerivedStateFromProps (../node_modules/react-dom/cjs/react-dom.development.js:12432:46)
      at mountClassInstance (../node_modules/react-dom/cjs/react-dom.development.js:12871:31)
      at updateClassComponent (../node_modules/react-dom/cjs/react-dom.development.js:17426:23)
      at beginWork (../node_modules/react-dom/cjs/react-dom.development.js:19073:16)
      at callCallback (../node_modules/react-dom/cjs/react-dom.development.js:3945:14)
      at invokeGuardedCallbackDev (../node_modules/react-dom/cjs/react-dom.development.js:3994:16)
      at invokeGuardedCallback (../node_modules/react-dom/cjs/react-dom.development.js:4056:31)
      at beginWork$1 (../node_modules/react-dom/cjs/react-dom.development.js:23959:28)
      at performUnitOfWork (../node_modules/react-dom/cjs/react-dom.development.js:22771:12)
      at workLoopSync (../node_modules/react-dom/cjs/react-dom.development.js:22702:22)
      at renderRootSync (../node_modules/react-dom/cjs/react-dom.development.js:22665:7)
      at performSyncWorkOnRoot (../node_modules/react-dom/cjs/react-dom.development.js:22288:18)
      at scheduleUpdateOnFiber (../node_modules/react-dom/cjs/react-dom.development.js:21876:28)
      at updateContainer (../node_modules/react-dom/cjs/react-dom.development.js:25477:24)
      at legacyRenderSubtreeIntoContainer/< (../node_modules/react-dom/cjs/react-dom.development.js:26016:22)
      at unbatchedUpdates (../node_modules/react-dom/cjs/react-dom.development.js:22426:12)
      at legacyRenderSubtreeIntoContainer (../node_modules/react-dom/cjs/react-dom.development.js:26015:21)
      at render (../node_modules/react-dom/cjs/react-dom.development.js:26098:10)
      at reactFirstRenderOrHydrate (webpack-internal:/.cache/app.js:79:22)
      at runRender (webpack-internal:/.cache/app.js:199:15)
      at  (webpack-internal:/.cache/app.js:212:9)
      at setTimeout handler* (webpack-internal:/.cache/app.js:211:17)
      at promise callback* (webpack-internal:/.cache/app.js:155:150)
      at promise callback* (webpack-internal:/.cache/app.js:96:86)
      at ./.cache/app.js (http:/localhost:8000/commons.js:2824:1)
      at options.factory (http:/localhost:8000/commons.js:8200:31)
      at __webpack_require__ (http:/localhost:8000/commons.js:7548:33)
      at __webpack_exports__< (http:/localhost:8000/commons.js:8934:130)
      at __webpack_require__.O (http:/localhost:8000/commons.js:7597:23)
      at  (http:/localhost:8000/commons.js:8935:53)
      at  (http:/localhost:8000/commons.js:8937:12)

  ● Drawer › open › have to match correct scroll view setup

    Transport not open

      at send (http:/localhost:8000/socket.io/socket.io.js:3135:15)
      at flush (http:/localhost:8000/socket.io/socket.io.js:2801:24)
      at sendPacket (http:/localhost:8000/socket.io/socket.io.js:2867:12)
      at write (http:/localhost:8000/socket.io/socket.io.js:2821:12)
      at _packet (http:/localhost:8000/socket.io/socket.io.js:598:21)
      at packet (http:/localhost:8000/socket.io/socket.io.js:1001:15)
      at emit (http:/localhost:8000/socket.io/socket.io.js:981:14)
      at unregisterPath (webpack-internal:/.cache/socketIo.js:63:10)
      at getDerivedStateFromProps (webpack-internal:/.cache/query-result-store.js:97:64)
      at applyDerivedStateFromProps (../node_modules/react-dom/cjs/react-dom.development.js:12432:46)
      at mountClassInstance (../node_modules/react-dom/cjs/react-dom.development.js:12871:31)
      at updateClassComponent (../node_modules/react-dom/cjs/react-dom.development.js:17426:23)
      at beginWork (../node_modules/react-dom/cjs/react-dom.development.js:19073:16)
      at callCallback (../node_modules/react-dom/cjs/react-dom.development.js:3945:14)
      at invokeGuardedCallbackDev (../node_modules/react-dom/cjs/react-dom.development.js:3994:16)
      at invokeGuardedCallback (../node_modules/react-dom/cjs/react-dom.development.js:4056:31)
      at beginWork$1 (../node_modules/react-dom/cjs/react-dom.development.js:23959:28)
      at performUnitOfWork (../node_modules/react-dom/cjs/react-dom.development.js:22771:12)
      at workLoopSync (../node_modules/react-dom/cjs/react-dom.development.js:22702:22)
      at renderRootSync (../node_modules/react-dom/cjs/react-dom.development.js:22665:7)
      at performSyncWorkOnRoot (../node_modules/react-dom/cjs/react-dom.development.js:22288:18)
      at scheduleUpdateOnFiber (../node_modules/react-dom/cjs/react-dom.development.js:21876:28)
      at updateContainer (../node_modules/react-dom/cjs/react-dom.development.js:25477:24)
      at legacyRenderSubtreeIntoContainer/< (../node_modules/react-dom/cjs/react-dom.development.js:26016:22)
      at unbatchedUpdates (../node_modules/react-dom/cjs/react-dom.development.js:22426:12)
      at legacyRenderSubtreeIntoContainer (../node_modules/react-dom/cjs/react-dom.development.js:26015:21)
      at render (../node_modules/react-dom/cjs/react-dom.development.js:26098:10)
      at reactFirstRenderOrHydrate (webpack-internal:/.cache/app.js:79:22)
      at runRender (webpack-internal:/.cache/app.js:199:15)
      at  (webpack-internal:/.cache/app.js:212:9)
      at setTimeout handler* (webpack-internal:/.cache/app.js:211:17)
      at promise callback* (webpack-internal:/.cache/app.js:155:150)
      at promise callback* (webpack-internal:/.cache/app.js:96:86)
      at ./.cache/app.js (http:/localhost:8000/commons.js:2824:1)
      at options.factory (http:/localhost:8000/commons.js:8200:31)
      at __webpack_require__ (http:/localhost:8000/commons.js:7548:33)
      at __webpack_exports__< (http:/localhost:8000/commons.js:8934:130)
      at __webpack_require__.O (http:/localhost:8000/commons.js:7597:23)
      at  (http:/localhost:8000/commons.js:8935:53)
```

Not sure "how critical" this is.

And one updated snapshot:
```
 PASS   browser: firefox  src/components/dropdown/__tests__/Dropdown.screenshot.test.ts (29.837 s)
 › 1 snapshot updated.
```

**Comment by langz** [comment]
> > @langz Could you retry it on your machine? Remember to run `yarn` (install) first.
> 
> I will give it a spin🕸️

After testing now, it seems much better here.

Here's the summary:
```
Snapshot Summary
 › 1 snapshot updated from 1 test suite.

Test Suites: 2 failed, 56 passed, 58 total
Tests:       2 failed, 461 passed, 463 total
Snapshots:   1 updated, 462 passed, 463 total
Time:        403.268 s, estimated 405 s
```


I do however seem to get two tests that fail with "Transport not open", as reported earlier:

```
 FAIL   browser: firefox  src/components/drawer/__tests__/Drawer.screenshot.test.ts (14.412 s)
  ● Drawer › open › have to match correct scroll view setup

    Transport not open

      at send (http:/localhost:8000/socket.io/socket.io.js:3135:15)
      at flush (http:/localhost:8000/socket.io/socket.io.js:2801:24)
      at sendPacket (http:/localhost:8000/socket.io/socket.io.js:2867:12)
      at write (http:/localhost:8000/socket.io/socket.io.js:2821:12)
      at _packet (http:/localhost:8000/socket.io/socket.io.js:598:21)
      at packet (http:/localhost:8000/socket.io/socket.io.js:1001:15)
      at emit (http:/localhost:8000/socket.io/socket.io.js:981:14)
      at unregisterPath (webpack-internal:/.cache/socketIo.js:63:10)
      at getDerivedStateFromProps (webpack-internal:/.cache/query-result-store.js:97:64)
      at applyDerivedStateFromProps (../node_modules/react-dom/cjs/react-dom.development.js:12432:46)
      at mountClassInstance (../node_modules/react-dom/cjs/react-dom.development.js:12871:31)
      at updateClassComponent (../node_modules/react-dom/cjs/react-dom.development.js:17426:23)
      at beginWork (../node_modules/react-dom/cjs/react-dom.development.js:19073:16)
      at callCallback (../node_modules/react-dom/cjs/react-dom.development.js:3945:14)
      at invokeGuardedCallbackDev (../node_modules/react-dom/cjs/react-dom.development.js:3994:16)
      at invokeGuardedCallback (../node_modules/react-dom/cjs/react-dom.development.js:4056:31)
      at beginWork$1 (../node_modules/react-dom/cjs/react-dom.development.js:23959:28)
      at performUnitOfWork (../node_modules/react-dom/cjs/react-dom.development.js:22771:12)
      at workLoopSync (../node_modules/react-dom/cjs/react-dom.development.js:22702:22)
      at renderRootSync (../node_modules/react-dom/cjs/react-dom.development.js:22665:7)
      at performSyncWorkOnRoot (../node_modules/react-dom/cjs/react-dom.development.js:22288:18)
      at scheduleUpdateOnFiber (../node_modules/react-dom/cjs/react-dom.development.js:21876:28)
      at updateContainer (../node_modules/react-dom/cjs/react-dom.development.js:25477:24)
      at legacyRenderSubtreeIntoContainer/< (../node_modules/react-dom/cjs/react-dom.development.js:26016:22)
      at unbatchedUpdates (../node_modules/react-dom/cjs/react-dom.development.js:22426:12)
      at legacyRenderSubtreeIntoContainer (../node_modules/react-dom/cjs/react-dom.development.js:26015:21)
      at render (../node_modules/react-dom/cjs/react-dom.development.js:26098:10)
      at reactFirstRenderOrHydrate (webpack-internal:/.cache/app.js:79:22)
      at runRender (webpack-internal:/.cache/app.js:199:15)
      at  (webpack-internal:/.cache/app.js:212:9)
      at setTimeout handler* (webpack-internal:/.cache/app.js:211:17)
      at promise callback* (webpack-internal:/.cache/app.js:155:150)
      at promise callback* (webpack-internal:/.cache/app.js:96:86)
      at ./.cache/app.js (http:/localhost:8000/commons.js:2824:1)
      at options.factory (http:/localhost:8000/commons.js:8200:31)
      at __webpack_require__ (http:/localhost:8000/commons.js:7548:33)
      at __webpack_exports__< (http:/localhost:8000/commons.js:8934:130)
      at __webpack_require__.O (http:/localhost:8000/commons.js:7597:23)
      at  (http:/localhost:8000/commons.js:8935:53)
      at  (http:/localhost:8000/commons.js:8937:12)

  ● Drawer › open › have to match correct scroll view setup

    Transport not open

      at send (http:/localhost:8000/socket.io/socket.io.js:3135:15)
      at flush (http:/localhost:8000/socket.io/socket.io.js:2801:24)
      at sendPacket (http:/localhost:8000/socket.io/socket.io.js:2867:12)
      at write (http:/localhost:8000/socket.io/socket.io.js:2821:12)
      at _packet (http:/localhost:8000/socket.io/socket.io.js:598:21)
      at packet (http:/localhost:8000/socket.io/socket.io.js:1001:15)
      at emit (http:/localhost:8000/socket.io/socket.io.js:981:14)
      at unregisterPath (webpack-internal:/.cache/socketIo.js:63:10)
      at getDerivedStateFromProps (webpack-internal:/.cache/query-result-store.js:97:64)
      at applyDerivedStateFromProps (../node_modules/react-dom/cjs/react-dom.development.js:12432:46)
      at mountClassInstance (../node_modules/react-dom/cjs/react-dom.development.js:12871:31)
      at updateClassComponent (../node_modules/react-dom/cjs/react-dom.development.js:17426:23)
      at beginWork (../node_modules/react-dom/cjs/react-dom.development.js:19073:16)
      at callCallback (../node_modules/react-dom/cjs/react-dom.development.js:3945:14)
      at invokeGuardedCallbackDev (../node_modules/react-dom/cjs/react-dom.development.js:3994:16)
      at invokeGuardedCallback (../node_modules/react-dom/cjs/react-dom.development.js:4056:31)
      at beginWork$1 (../node_modules/react-dom/cjs/react-dom.development.js:23959:28)
      at performUnitOfWork (../node_modules/react-dom/cjs/react-dom.development.js:22771:12)
      at workLoopSync (../node_modules/react-dom/cjs/react-dom.development.js:22702:22)
      at renderRootSync (../node_modules/react-dom/cjs/react-dom.development.js:22665:7)
      at performSyncWorkOnRoot (../node_modules/react-dom/cjs/react-dom.development.js:22288:18)
      at scheduleUpdateOnFiber (../node_modules/react-dom/cjs/react-dom.development.js:21876:28)
      at updateContainer (../node_modules/react-dom/cjs/react-dom.development.js:25477:24)
      at legacyRenderSubtreeIntoContainer/< (../node_modules/react-dom/cjs/react-dom.development.js:26016:22)
      at unbatchedUpdates (../node_modules/react-dom/cjs/react-dom.development.js:22426:12)
      at legacyRenderSubtreeIntoContainer (../node_modules/react-dom/cjs/react-dom.development.js:26015:21)
      at render (../node_modules/react-dom/cjs/react-dom.development.js:26098:10)
      at reactFirstRenderOrHydrate (webpack-internal:/.cache/app.js:79:22)
      at runRender (webpack-internal:/.cache/app.js:199:15)
      at  (webpack-internal:/.cache/app.js:212:9)
      at setTimeout handler* (webpack-internal:/.cache/app.js:211:17)
      at promise callback* (webpack-internal:/.cache/app.js:155:150)
      at promise callback* (webpack-internal:/.cache/app.js:96:86)
      at ./.cache/app.js (http:/localhost:8000/commons.js:2824:1)
      at options.factory (http:/localhost:8000/commons.js:8200:31)
      at __webpack_require__ (http:/localhost:8000/commons.js:7548:33)
      at __webpack_exports__< (http:/localhost:8000/commons.js:8934:130)
      at __webpack_require__.O (http:/localhost:8000/commons.js:7597:23)
      at  (http:/localhost:8000/commons.js:8935:53)
```

Not sure "how critical" this is.

And one updated snapshot:
```
 PASS   browser: firefox  src/components/dropdown/__tests__/Dropdown.screenshot.test.ts (29.837 s)
 › 1 snapshot updated.
```

**Comment by langz** [comment]
> > @langz Could you retry it on your machine? Remember to run `yarn` (install) first.
> 
> I will give it a spin🕸️

After testing now, it seems much better here.

Here's the summary:
```
Snapshot Summary
 › 1 snapshot updated from 1 test suite.

Test Suites: 2 failed, 56 passed, 58 total
Tests:       2 failed, 461 passed, 463 total
Snapshots:   1 updated, 462 passed, 463 total
Time:        403.268 s, estimated 405 s
```


I do however seem to get two tests that fail with "Transport not open", as reported earlier:

```
 FAIL   browser: firefox  src/components/drawer/__tests__/Drawer.screenshot.test.ts (14.412 s)
  ● Drawer › open › have to match correct scroll view setup

    Transport not open

      at send (http:/localhost:8000/socket.io/socket.io.js:3135:15)
      at flush (http:/localhost:8000/socket.io/socket.io.js:2801:24)
      at sendPacket (http:/localhost:8000/socket.io/socket.io.js:2867:12)
      at write (http:/localhost:8000/socket.io/socket.io.js:2821:12)
      at _packet (http:/localhost:8000/socket.io/socket.io.js:598:21)
      at packet (http:/localhost:8000/socket.io/socket.io.js:1001:15)
      at emit (http:/localhost:8000/socket.io/socket.io.js:981:14)
      at unregisterPath (webpack-internal:/.cache/socketIo.js:63:10)
      at getDerivedStateFromProps (webpack-internal:/.cache/query-result-store.js:97:64)
      at applyDerivedStateFromProps (../node_modules/react-dom/cjs/react-dom.development.js:12432:46)
      at mountClassInstance (../node_modules/react-dom/cjs/react-dom.development.js:12871:31)
      at updateClassComponent (../node_modules/react-dom/cjs/react-dom.development.js:17426:23)
      at beginWork (../node_modules/react-dom/cjs/react-dom.development.js:19073:16)
      at callCallback (../node_modules/react-dom/cjs/react-dom.development.js:3945:14)
      at invokeGuardedCallbackDev (../node_modules/react-dom/cjs/react-dom.development.js:3994:16)
      at invokeGuardedCallback (../node_modules/react-dom/cjs/react-dom.development.js:4056:31)
      at beginWork$1 (../node_modules/react-dom/cjs/react-dom.development.js:23959:28)
      at performUnitOfWork (../node_modules/react-dom/cjs/react-dom.development.js:22771:12)
      at workLoopSync (../node_modules/react-dom/cjs/react-dom.development.js:22702:22)
      at renderRootSync (../node_modules/react-dom/cjs/react-dom.development.js:22665:7)
      at performSyncWorkOnRoot (../node_modules/react-dom/cjs/react-dom.development.js:22288:18)
      at scheduleUpdateOnFiber (../node_modules/react-dom/cjs/react-dom.development.js:21876:28)
      at updateContainer (../node_modules/react-dom/cjs/react-dom.development.js:25477:24)
      at legacyRenderSubtreeIntoContainer/< (../node_modules/react-dom/cjs/react-dom.development.js:26016:22)
      at unbatchedUpdates (../node_modules/react-dom/cjs/react-dom.development.js:22426:12)
      at legacyRenderSubtreeIntoContainer (../node_modules/react-dom/cjs/react-dom.development.js:26015:21)
      at render (../node_modules/react-dom/cjs/react-dom.development.js:26098:10)
      at reactFirstRenderOrHydrate (webpack-internal:/.cache/app.js:79:22)
      at runRender (webpack-internal:/.cache/app.js:199:15)
      at  (webpack-internal:/.cache/app.js:212:9)
      at setTimeout handler* (webpack-internal:/.cache/app.js:211:17)
      at promise callback* (webpack-internal:/.cache/app.js:155:150)
      at promise callback* (webpack-internal:/.cache/app.js:96:86)
      at ./.cache/app.js (http:/localhost:8000/commons.js:2824:1)
      at options.factory (http:/localhost:8000/commons.js:8200:31)
      at __webpack_require__ (http:/localhost:8000/commons.js:7548:33)
      at __webpack_exports__< (http:/localhost:8000/commons.js:8934:130)
      at __webpack_require__.O (http:/localhost:8000/commons.js:7597:23)
      at  (http:/localhost:8000/commons.js:8935:53)
      at  (http:/localhost:8000/commons.js:8937:12)

  ● Drawer › open › have to match correct scroll view setup

    Transport not open

      at send (http:/localhost:8000/socket.io/socket.io.js:3135:15)
      at flush (http:/localhost:8000/socket.io/socket.io.js:2801:24)
      at sendPacket (http:/localhost:8000/socket.io/socket.io.js:2867:12)
      at write (http:/localhost:8000/socket.io/socket.io.js:2821:12)
      at _packet (http:/localhost:8000/socket.io/socket.io.js:598:21)
      at packet (http:/localhost:8000/socket.io/socket.io.js:1001:15)
      at emit (http:/localhost:8000/socket.io/socket.io.js:981:14)
      at unregisterPath (webpack-internal:/.cache/socketIo.js:63:10)
      at getDerivedStateFromProps (webpack-internal:/.cache/query-result-store.js:97:64)
      at applyDerivedStateFromProps (../node_modules/react-dom/cjs/react-dom.development.js:12432:46)
      at mountClassInstance (../node_modules/react-dom/cjs/react-dom.development.js:12871:31)
      at updateClassComponent (../node_modules/react-dom/cjs/react-dom.development.js:17426:23)
      at beginWork (../node_modules/react-dom/cjs/react-dom.development.js:19073:16)
      at callCallback (../node_modules/react-dom/cjs/react-dom.development.js:3945:14)
      at invokeGuardedCallbackDev (../node_modules/react-dom/cjs/react-dom.development.js:3994:16)
      at invokeGuardedCallback (../node_modules/react-dom/cjs/react-dom.development.js:4056:31)
      at beginWork$1 (../node_modules/react-dom/cjs/react-dom.development.js:23959:28)
      at performUnitOfWork (../node_modules/react-dom/cjs/react-dom.development.js:22771:12)
      at workLoopSync (../node_modules/react-dom/cjs/react-dom.development.js:22702:22)
      at renderRootSync (../node_modules/react-dom/cjs/react-dom.development.js:22665:7)
      at performSyncWorkOnRoot (../node_modules/react-dom/cjs/react-dom.development.js:22288:18)
      at scheduleUpdateOnFiber (../node_modules/react-dom/cjs/react-dom.development.js:21876:28)
      at updateContainer (../node_modules/react-dom/cjs/react-dom.development.js:25477:24)
      at legacyRenderSubtreeIntoContainer/< (../node_modules/react-dom/cjs/react-dom.development.js:26016:22)
      at unbatchedUpdates (../node_modules/react-dom/cjs/react-dom.development.js:22426:12)
      at legacyRenderSubtreeIntoContainer (../node_modules/react-dom/cjs/react-dom.development.js:26015:21)
      at render (../node_modules/react-dom/cjs/react-dom.development.js:26098:10)
      at reactFirstRenderOrHydrate (webpack-internal:/.cache/app.js:79:22)
      at runRender (webpack-internal:/.cache/app.js:199:15)
      at  (webpack-internal:/.cache/app.js:212:9)
      at setTimeout handler* (webpack-internal:/.cache/app.js:211:17)
      at promise callback* (webpack-internal:/.cache/app.js:155:150)
      at promise callback* (webpack-internal:/.cache/app.js:96:86)
      at ./.cache/app.js (http:/localhost:8000/commons.js:2824:1)
      at options.factory (http:/localhost:8000/commons.js:8200:31)
      at __webpack_require__ (http:/localhost:8000/commons.js:7548:33)
      at __webpack_exports__< (http:/localhost:8000/commons.js:8934:130)
      at __webpack_require__.O (http:/localhost:8000/commons.js:7597:23)
      at  (http:/localhost:8000/commons.js:8935:53)
```

Not sure "how critical" this is.

And one updated snapshot:
```
 PASS   browser: firefox  src/components/dropdown/__tests__/Dropdown.screenshot.test.ts (29.837 s)
 › 1 snapshot updated.
```

**Comment by langz** [comment]
After testing now, it seems much better here.

Here's the summary:
```
Snapshot Summary
 › 1 snapshot updated from 1 test suite.

Test Suites: 2 failed, 56 passed, 58 total
Tests:       2 failed, 461 passed, 463 total
Snapshots:   1 updated, 462 passed, 463 total
Time:        403.268 s, estimated 405 s
```


I do however seem to get two tests that fail with "Transport not open", as reported earlier:

```
 FAIL   browser: firefox  src/components/drawer/__tests__/Drawer.screenshot.test.ts (14.412 s)
  ● Drawer › open › have to match correct scroll view setup

    Transport not open

      at send (http:/localhost:8000/socket.io/socket.io.js:3135:15)
      at flush (http:/localhost:8000/socket.io/socket.io.js:2801:24)
      at sendPacket (http:/localhost:8000/socket.io/socket.io.js:2867:12)
      at write (http:/localhost:8000/socket.io/socket.io.js:2821:12)
      at _packet (http:/localhost:8000/socket.io/socket.io.js:598:21)
      at packet (http:/localhost:8000/socket.io/socket.io.js:1001:15)
      at emit (http:/localhost:8000/socket.io/socket.io.js:981:14)
      at unregisterPath (webpack-internal:/.cache/socketIo.js:63:10)
      at getDerivedStateFromProps (webpack-internal:/.cache/query-result-store.js:97:64)
      at applyDerivedStateFromProps (../node_modules/react-dom/cjs/react-dom.development.js:12432:46)
      at mountClassInstance (../node_modules/react-dom/cjs/react-dom.development.js:12871:31)
      at updateClassComponent (../node_modules/react-dom/cjs/react-dom.development.js:17426:23)
      at beginWork (../node_modules/react-dom/cjs/react-dom.development.js:19073:16)
      at callCallback (../node_modules/react-dom/cjs/react-dom.development.js:3945:14)
      at invokeGuardedCallbackDev (../node_modules/react-dom/cjs/react-dom.development.js:3994:16)
      at invokeGuardedCallback (../node_modules/react-dom/cjs/react-dom.development.js:4056:31)
      at beginWork$1 (../node_modules/react-dom/cjs/react-dom.development.js:23959:28)
      at performUnitOfWork (../node_modules/react-dom/cjs/react-dom.development.js:22771:12)
      at workLoopSync (../node_modules/react-dom/cjs/react-dom.development.js:22702:22)
      at renderRootSync (../node_modules/react-dom/cjs/react-dom.development.js:22665:7)
      at performSyncWorkOnRoot (../node_modules/react-dom/cjs/react-dom.development.js:22288:18)
      at scheduleUpdateOnFiber (../node_modules/react-dom/cjs/react-dom.development.js:21876:28)
      at updateContainer (../node_modules/react-dom/cjs/react-dom.development.js:25477:24)
      at legacyRenderSubtreeIntoContainer/< (../node_modules/react-dom/cjs/react-dom.development.js:26016:22)
      at unbatchedUpdates (../node_modules/react-dom/cjs/react-dom.development.js:22426:12)
      at legacyRenderSubtreeIntoContainer (../node_modules/react-dom/cjs/react-dom.development.js:26015:21)
      at render (../node_modules/react-dom/cjs/react-dom.development.js:26098:10)
      at reactFirstRenderOrHydrate (webpack-internal:/.cache/app.js:79:22)
      at runRender (webpack-internal:/.cache/app.js:199:15)
      at  (webpack-internal:/.cache/app.js:212:9)
      at setTimeout handler* (webpack-internal:/.cache/app.js:211:17)
      at promise callback* (webpack-internal:/.cache/app.js:155:150)
      at promise callback* (webpack-internal:/.cache/app.js:96:86)
      at ./.cache/app.js (http:/localhost:8000/commons.js:2824:1)
      at options.factory (http:/localhost:8000/commons.js:8200:31)
      at __webpack_require__ (http:/localhost:8000/commons.js:7548:33)
      at __webpack_exports__< (http:/localhost:8000/commons.js:8934:130)
      at __webpack_require__.O (http:/localhost:8000/commons.js:7597:23)
      at  (http:/localhost:8000/commons.js:8935:53)
      at  (http:/localhost:8000/commons.js:8937:12)

  ● Drawer › open › have to match correct scroll view setup

    Transport not open

      at send (http:/localhost:8000/socket.io/socket.io.js:3135:15)
      at flush (http:/localhost:8000/socket.io/socket.io.js:2801:24)
      at sendPacket (http:/localhost:8000/socket.io/socket.io.js:2867:12)
      at write (http:/localhost:8000/socket.io/socket.io.js:2821:12)
      at _packet (http:/localhost:8000/socket.io/socket.io.js:598:21)
      at packet (http:/localhost:8000/socket.io/socket.io.js:1001:15)
      at emit (http:/localhost:8000/socket.io/socket.io.js:981:14)
      at unregisterPath (webpack-internal:/.cache/socketIo.js:63:10)
      at getDerivedStateFromProps (webpack-internal:/.cache/query-result-store.js:97:64)
      at applyDerivedStateFromProps (../node_modules/react-dom/cjs/react-dom.development.js:12432:46)
      at mountClassInstance (../node_modules/react-dom/cjs/react-dom.development.js:12871:31)
      at updateClassComponent (../node_modules/react-dom/cjs/react-dom.development.js:17426:23)
      at beginWork (../node_modules/react-dom/cjs/react-dom.development.js:19073:16)
      at callCallback (../node_modules/react-dom/cjs/react-dom.development.js:3945:14)
      at invokeGuardedCallbackDev (../node_modules/react-dom/cjs/react-dom.development.js:3994:16)
      at invokeGuardedCallback (../node_modules/react-dom/cjs/react-dom.development.js:4056:31)
      at beginWork$1 (../node_modules/react-dom/cjs/react-dom.development.js:23959:28)
      at performUnitOfWork (../node_modules/react-dom/cjs/react-dom.development.js:22771:12)
      at workLoopSync (../node_modules/react-dom/cjs/react-dom.development.js:22702:22)
      at renderRootSync (../node_modules/react-dom/cjs/react-dom.development.js:22665:7)
      at performSyncWorkOnRoot (../node_modules/react-dom/cjs/react-dom.development.js:22288:18)
      at scheduleUpdateOnFiber (../node_modules/react-dom/cjs/react-dom.development.js:21876:28)
      at updateContainer (../node_modules/react-dom/cjs/react-dom.development.js:25477:24)
      at legacyRenderSubtreeIntoContainer/< (../node_modules/react-dom/cjs/react-dom.development.js:26016:22)
      at unbatchedUpdates (../node_modules/react-dom/cjs/react-dom.development.js:22426:12)
      at legacyRenderSubtreeIntoContainer (../node_modules/react-dom/cjs/react-dom.development.js:26015:21)
      at render (../node_modules/react-dom/cjs/react-dom.development.js:26098:10)
      at reactFirstRenderOrHydrate (webpack-internal:/.cache/app.js:79:22)
      at runRender (webpack-internal:/.cache/app.js:199:15)
      at  (webpack-internal:/.cache/app.js:212:9)
      at setTimeout handler* (webpack-internal:/.cache/app.js:211:17)
      at promise callback* (webpack-internal:/.cache/app.js:155:150)
      at promise callback* (webpack-internal:/.cache/app.js:96:86)
      at ./.cache/app.js (http:/localhost:8000/commons.js:2824:1)
      at options.factory (http:/localhost:8000/commons.js:8200:31)
      at __webpack_require__ (http:/localhost:8000/commons.js:7548:33)
      at __webpack_exports__< (http:/localhost:8000/commons.js:8934:130)
      at __webpack_require__.O (http:/localhost:8000/commons.js:7597:23)
      at  (http:/localhost:8000/commons.js:8935:53)
```

Not sure "how critical" this is.

And one updated snapshot:
```
 PASS   browser: firefox  src/components/dropdown/__tests__/Dropdown.screenshot.test.ts (29.837 s)
 › 1 snapshot updated.
```

**Comment by tujoworker** [comment]
> Transport not open

Regarding this issue, could you pull the latest and run yarn install, and try again?

I hope its about [this issue](https://github.com/socketio/socket.io/issues/3838).

**Review by langz — APPROVED** [review]

File: packages/dnb-design-system-portal/src/docs/contribute/faq.mdx [hunk 1]
```diff
@@ -44,12 +44,50 @@ When switching over to Yarn PnP, there are some issues:
...
+
+```json
+{
+  "resolutions": {
+    "gatsby/browserslist": "4.21.4"
+  }
+}
+```
+
+Gatsby v4 uses socket.io v3 that has en [issue](https://github.com/socketio/socket.io/issues/3838) when running all visual tests in a batch locally on a development build. To mitigate it, we have installed a newer version of socket.io:
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/contribute/faq.mdx:59]:
```suggestion
Gatsby v4 uses socket.io v3 that has an [issue](https://github.com/socketio/socket.io/issues/3838) when running all visual tests in a batch locally on a development build. To mitigate it, we have installed a newer version of socket.io:
```

File: packages/dnb-design-system-portal/src/docs/contribute/faq.mdx [hunk 2]
```diff
@@ -44,12 +44,50 @@ When switching over to Yarn PnP, there are some issues:
...
+
+## MDX v2
 
 In MDX v1 the Portal used an MDX template (`./templates/mdx.js`) with the `createPage` API. This is currently not the case anymore.
 Instead, the "PortalLayout" is used to wrap `.mdx` pages inside the `wrapPageElement` API.
 This is due to the `JavaScript heap out of memory` error when using templates.
 
+## Visual tests and Playwright
+
+Jest does start several works at the same time. When we simulate states on e.g. ToggleButton and Button – Playwright (v1.31.2) struggles to handle this. Some of the hover or focus tests will fail. To ensure Jest we never execute several workers as the same time, we set `--maxWorkers=1`.
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/contribute/faq.mdx:89]:
```suggestion
Jest does start several workers at the same time. When we simulate states on e.g. ToggleButton and Button – Playwright (v1.31.2) struggles to handle this. Some of the hover or focus tests will fail. To ensure Jest never execute several workers as the same time, we set `--maxWorkers=1`.
```

**Review by langz — APPROVED** [review]

**Pull request merged**